services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankealueEntity.kt

```diff
@@ -50,6 +56,15 @@ class HankealueEntity(
         orphanRemoval = true
     )
     var tormaystarkasteluTulos: TormaystarkasteluTulosEntity?,
+    @ElementCollection(fetch = FetchType.LAZY)
+    @CollectionTable(
+        name = "hankkeen_haittojenhallintasuunnitelma",
+        joinColumns = [JoinColumn(name = "hankealue_id", referencedColumnName = "id")]
+    )
+    @MapKeyColumn(name = "tyyppi")
+    @Column(name = "sisalto")
+    @MapKeyEnumerated(EnumType.STRING)
+    var haittojenhallintasuunnitelma: Map<Haittojenhallintatyyppi, String>? = null
```

Comment:
Please add a trailing comma.

Reply:
Is there a difference between an empty map and a null map? Could this be non-nullable with a default of an empty map? Or a non-nullable with a default value?

Reply:
Added.